data/list/basic.lean

```diff
@@ -957,6 +957,32 @@ theorem drop_eq_nth_le_cons : ∀ {n} {l : list α} h,
 | 0     (a::l) h := rfl
 | (n+1) (a::l) h := @drop_eq_nth_le_cons n _ _
 
+/- simp? -/
+theorem drop_drop : ∀ (n m) (l : list α), drop n (drop m l) = drop (n + m) l
```

Comment:
yes, this looks like a good rewrite rule.
Mark it `@[simp]`

Reply:
Done.